packages/flutter/lib/src/widgets/basic.dart

```diff
@@ -5832,6 +5835,22 @@ class MouseRegion extends SingleChildRenderObjectWidget {
   /// the pointer.
   final PointerExitEventListener onExit;
 
+  /// Whether this widget should prevent other [MouseRegion]s visually behind it
+  /// from detecting the pointer, thus affecting how their [onHover], [onEnter],
+  /// and [onExit] behave.
+  ///
+  /// If [opaque] is true, this widget will absorb the mouse pointer, and
+  /// prevents this widget's siblings (or any other widgets that are not
+  /// ancestors or descendants of this widgets) from detecting the mouse
```

Comment:
```suggestion
  /// ancestors or descendants of this widget) from detecting the mouse
```
or
```suggestion
  /// ancestors or descendants of these widgets) from detecting the mouse
```
depending on meaning.